schemas/aerleon-policies.schema.json

```diff
@@ -33,30 +46,46 @@
       }
     },
     "filter": {
-      "type": "object",
-      "required": ["header", "terms"],
-      "properties": {
-        "header": {
+      "oneOf": [
+        {
           "type": "object",
-          "title": "Filter Header",
-          "description": "The filter header lists the generator target platforms and their options. Other filter-level settings can be defined in the header.",
-          "required": ["targets"],
+          "required": ["header", "terms"],
           "properties": {
-            "targets": {
-              "title": "Filter Targets",
-              "description": "aclgen will generate an acl for each target platforms identified here.",
-              "type": "object"
+            "header": {
+              "type": "object",
+              "title": "Filter Header",
+              "description": "The filter header lists the generator target platforms and their options. Other filter-level settings can be defined in the header.",
+              "required": ["targets"],
+              "properties": {
+                "targets": {
+                  "title": "Filter Targets",
+                  "description": "aclgen will generate an acl for each target platforms identified here.",
```

Comment:
s/aclgen/Aclgen?

Reply:
aclgen is always lowercase because it is a command line program. We never refer to it as Aclgen even in docs or errors even at the start of sentences. But ACL should be upper case.